pkg/chunk/gcp/storage_client.go

```diff
@@ -116,6 +117,17 @@ func (s *storageClient) QueryPages(ctx context.Context, query chunk.IndexQuery,
 	table := s.client.Open(query.TableName)
 
 	var rowRange bigtable.RowRange
+
+	/* BigTable only seems to support regex match on cell values, so doing it
+	   client side for now
+	readOpts := []bigtable.ReadOption{
```

Comment:
I'm unclear why this code is commented out.  Is it an example of what you'd like to do but doesn't work?

Reply:
Yeah, pretty much - I'm hoping to get some input from the Bigtable team on this.  Seems weird we can do regex matches on values, but not equality matches.

Normally don't like to leave this kind of mess in there, and can remove it if you like.  I'll move it to another PR that will sit open otherwise though - I need somewhere to point them at.